binance_trade_bot/database.py

```diff
@@ -117,6 +117,15 @@ def get_current_coin(self) -> Optional[Coin]:
             session.expunge(coin)
             return coin
 
+    def get_current_coin_date(self) -> Optional[datetime]:
+        session: Session
+        with self.db_session() as session:
+            current_coin = session.query(CurrentCoin).order_by(CurrentCoin.datetime.desc()).first()
+            if current_coin is None:
+                return datetime.now()
+            coin_date = current_coin.datetime
```

Comment:
[SQLAlchemy docs on Expunging](https://docs.sqlalchemy.org/en/13/orm/session_state_management.html#expunging)
Can't say I really understand myself. It sounds like it makes SQLAlchemy stop tracking it, so you just have the barebones object.

Don't think you need it here since you're returning the datetime, not the model.

Reply:
cool okay that makes sense, thanks!